cmd/translate.go

```diff
@@ -0,0 +1,17 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var translateCmd = &cobra.Command{
+	Use:   "translate",
+	Short: "This command sets configuration",
+	Long: `translate command sets configuration from:
+			environment variables
+			yaml file
+			CLI arguments`,
```

Comment:
root.go is responsible for setting config

Reply:
I'm not sure. I just wrote description according to the task https://github.com/translate-agent/translate/issues/6 where is written that translate cmd should be able to set config but in any case we can change the descriptions if needed.